<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormWgtMsgBox_H</class>
 <widget class="QWidget" name="FormWgtMsgBox_H">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>196</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>196</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>196</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background:transparent;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="Body" native="true">
     <property name="maximumSize">
      <size>
       <width>360</width>
       <height>196</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#Body{
    border-radius:0px;
   color: rgb(46, 253, 251);
   border-image: url(:/image/door.png);
}


</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="MScrollArea" name="scrollArea">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QScrollArea{background-color:transparent;}
QListView{background:#5555ff;border-radius:0px; border: 0px solid gray;}
QListView::item:focus{border-radius:0px; border: 5px solid rgb(255, 128, 0);outline: none;}
QScrollBar:vertical {border:0px solid rgb(52, 72, 105);background:rgb(52, 72, 105);width:25px;margin:30 0 30 0px;}
QScrollBar::handle:vertical{ background: rgb(41, 142, 214);border-radius:5px;width:10px; height: 10px ; }
QScrollBar::add-line:vertical {border:1px solid rgb(52, 134, 208);background:rgb(52, 134, 208);height:30px;subcontrol-position:bottom;subcontrol-origin:margin;}
QScrollBar::sub-line:vertical {border:1px solid rgb(52, 134, 208);background:rgb(52, 134, 208);height:30px;subcontrol-position:top;subcontrol-origin:margin;}
QScrollBar::down-arrow:vertical {width:25px;height:25px; border-image:url(:/image/down_Arrow_2.png);}
QScrollBar::up-arrow:vertical {width:25px;height:25px;border-image:url(:/image/up_Arrow_1.png);}
QScrollBar::add-page:vertical {background:rgb(52, 72, 105);}
QScrollBar::sub-page:vertical {background:rgb(52, 72, 105);}
</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="wgt">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>340</width>
           <height>120</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background:transparent;</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="lblMessage">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:  #FFFFFF;
 </string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="Buttons" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnCancel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton
{
   background-color: transparent;
	border:0px solid gray;
	border-radius:0px;
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../resource/image/image.qrc">
             <normaloff>:/image/cancel.png</normaloff>:/image/cancel.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>56</width>
             <height>56</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnOk">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton
{
   background-color: transparent;
	border:0px solid gray;
	border-radius:0px;
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../resource/image/image.qrc">
             <normaloff>:/image/ok.png</normaloff>:/image/ok.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>56</width>
             <height>56</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MScrollArea</class>
   <extends>QScrollArea</extends>
   <header location="global">mscrollarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource/image/image.qrc"/>
 </resources>
 <connections/>
</ui>
